### DISCUSSION TITLE
Remap fail

### DISCUSSION THREAD

**Discussion by uPagge** [umputun/remark42#1606] [Q&A] [answered]
Maybe I'm doing something wrong, but when I run remap, I get the following error:

```
docker exec -it container-remark-name remap -s site_id -f /srv/var/
OCI runtime exec failed: exec failed: unable to start container process: exec: "remap": executable file not found in $PATH: unknown
```

**Comment by uPagge** [comment]
So, I figured it out, the team should look something like this

```
docker exec -it remark-container-name remark42 remap -s site_id -f /srv/var/rules
```

But I still get a mistake. ADMIN_PASSWD is installed

```
remark42 v1.11.3-8357846-20230121T13:27:39
2023/03/08 08:19:34.644 [INFO]  start remap, site struchkov, file with rules /srv/var/rules
2023/03/08 08:19:34.717 [ERROR] failed with error response "401 Unauthorized", Unauthorized
2023/03/08 08:19:34.717 [ERROR] failed with error response "401 Unauthorized", Unauthorized
error response "401 Unauthorized", Unauthorized
```

**Comment by uPagge** [comment, marked as answer]
So, I panicked early. This problem went away with compose down compose up.

But it's probably worth updating the documentation :)

You can also add an undocumented feature with substitution * there, so that you do not have to prescribe each URL. I accidentally noticed it in the PR of this revision.

**Reply by paskal** [reply]
Can you please update the doc with your learnings? Would be amazing!

**Reply by uPagge** [reply]
I'll try
